content/docs/started/getting-started-docker-desktop.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Getting Started with Kubeflow on Docker Desktop"
+weight = 10
++++
+
+This document outlines the steps to get Kubeflow running with Docker Desktop. Docker Desktop allows you to build production ready applications locally. 
+
+By the end of this tutorial, you should be able to get Kubeflow running and access the Kubeflow Dashboard.
+
+Pre-requisites:
+
+- Laptop, Desktop or a Workstation
+- Mac OS X or Windows 10
+- Docker Desktop (https://www.docker.com/products/docker-desktop) for Mac or Windows
+- sudo or admin access on the local machine
+- Access to an Internet connection with reasonable bandwidth
+
+# Enable Kubernetes 
+
+1. Click on the Docker Desktop logo and select 'Preferences...' 
+2. On the 'Kubernetes' tab tick the 'Enable Kubernetes' checkbox and then click on 'Apply'. 
+
+# Install kubectl
+
+```bash
+$ brew install kubectl
```

Comment:
I don't think we can use brew since that's macOS only. We can probably assume it's installed, but link to the docs if not?